dangerfile.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+
+// const { danger, fail, warn } = require('danger');
```

Comment:
Waiting on a release with https://github.com/danger/danger-js/pull/82 to support having this uncommented ( Danger normally comments out these requires and injects the deps in at runtime. It's only there for tooling support )